<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADWIF - Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Simulation">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="action_Reload"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Inspector"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Map">
    <property name="title">
     <string>&amp;Map</string>
    </property>
    <addaction name="action_Create_Map"/>
    <addaction name="action_Load_Map"/>
    <addaction name="action_Unload_Map"/>
    <addaction name="separator"/>
    <addaction name="action_Terrain_Parameters"/>
   </widget>
   <addaction name="menu_Simulation"/>
   <addaction name="menu_Map"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockInspector">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QtTreePropertyBrowser" name="propertyBrowser" native="true"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QFrame" name="frameActions">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>143</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="textCurrentSelection">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Nothing selected</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonCopy">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Copy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buttonEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Edit..</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="buttonExport">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Export..</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="buttonReset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Reset</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="buttonStatistics">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Statistics..</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buttonGraphs">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Graphs..</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableLog">
         <property name="styleSheet">
          <string notr="true">QTableWidget::item { padding: 0px }</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Source</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Message</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Reload">
   <property name="text">
    <string>&amp;Reload Data</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export..</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Inspector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Inspector</string>
   </property>
  </action>
  <action name="action_Terrain_Parameters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edit Parameters..</string>
   </property>
  </action>
  <action name="action_Create_Map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Create</string>
   </property>
  </action>
  <action name="action_Load_Map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Unload_Map">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Desktop/dev/ADWIF/editor/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Inspector</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockInspector</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
